modules/Core/classes/Recaptcha2.php

```diff
@@ -0,0 +1,44 @@
+<?php
+class Recaptcha2 extends CaptchaBase {
```

Comment:
Missing title comment, not sure if this is intentional

Reply:
It was intentional, I'll add them in though for consistency